accounting/src/test/java/io/skai/accounting/cucumber/steps/ClientCreationSteps.java

```diff
@@ -0,0 +1,78 @@
+package io.skai.accounting.cucumber.steps;
+
+import groovy.util.logging.Slf4j;
+import io.cucumber.java.After;
+import io.cucumber.java.DataTableType;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.skai.accounting.cucumber.CucumberBootstrap;
+import io.skai.accounting.dto.client.ClientDto;
+import io.skai.accounting.dto.client.ClientRequestDto;
+import io.skai.accounting.service.ClientService;
+import org.jooq.DSLContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Map;
+
+import static io.skai.accounting.jooq.Tables.CLIENT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+public class ClientCreationSteps extends CucumberBootstrap {
+
+    @Autowired
+    private DSLContext dslContext;
+
+    @Autowired
+    private ClientService clientService;
+
+    @Given("^Client absent in db")
```

Comment:
You truncate DB so better to be more precese in step name - like Cleanup test env, you can do multiple steps in env cleanup step def

Reply:
I used 2 times truncate for next reasons: in given i cleaned to be sure that test table is clear, in after - because its good practice to put test env in state that was before tests. i will delete given step, because in in normal situation table "clients" have to be empty and cleaned after other tests.